app/src/auth/utils/loginInterceptor.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Redirect to login when axios calls fail to authenticate.
+ */
+
+import axios from 'axios'
+import routes from '$routes'
+
+const meURL = `${process.env.STREAMR_API_URL}/users/me`
```

Comment:
Let's wrap API urls with `formatApiUrl('users', 'me')`

Reply:
Done in eea635e